### PR TITLE
Implements @ignite_versions decorator. Add support to override parame…

### PR DESCRIPTION
**Pull request by ivandasch**
…ters through --globals ignite_versions

Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/ducktests/tests/ignitetest/tests/add_node_rebalance_test.py [hunk 1]
```diff
@@ -39,20 +39,17 @@ class AddNodeRebalanceTest(IgniteTest):
     REBALANCE_TIMEOUT = 60
 
     @cluster(num_nodes=NUM_NODES + 1)
-    @parametrize(version=str(DEV_BRANCH))
-    @parametrize(version=str(LATEST))
-    def test_add_node(self, version):
+    @ignite_versions(str(DEV_BRANCH), str(LATEST))
```
Comment (timoninmaxim) [inline modules/ducktests/tests/ignitetest/tests/add_node_rebalance_test.py:42]: As we introduce own decorator to work with versions - let's remove requirement to use `str` for versions. It's ugly

Comment (timoninmaxim) [inline modules/ducktests/tests/ignitetest/tests/add_node_rebalance_test.py:42]: As we introduce own decorator to work with versions - let's remove requirement to use str for versions. It's ugly

**Review by timoninmaxim — CHANGES_REQUESTED** [review]

File: modules/ducktests/tests/ignitetest/tests/add_node_rebalance_test.py (hunk 1 above)
Comment (ivandasch) [reply modules/ducktests/tests/ignitetest/tests/add_node_rebalance_test.py:42]: Unfortunatelly, args should be json serializable. I'm afraid that it is not possible.

Comment (ivandasch) [reply modules/ducktests/tests/ignitetest/tests/add_node_rebalance_test.py:42]: It is possible after one bug in ducktape is resolved. I'll fix it ASAP, then I will rewrite it

**Pull request merged**